include/z64math.h

```diff
@@ -7,39 +7,39 @@
 
 typedef struct {
     f32 x, y;
-} Vec2f;
+} Vec2f; //size = 0x08
```

Comment:
This suggestion applies to all size comments:
```suggestion
} Vec2f; // size = 0x08
```